Bottom: b23ee80a95d8243de3149ae6c4f0d29d8ae74941
Top:    7461373a2821d207a4b06a1d482941e7ee5fb665
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-03-05 11:07:44 +1300

Refresh of threadsafestreamresult

---

diff --git a/testtools/tests/test_testsuite.py b/testtools/tests/test_testsuite.py
index 57f79f9..90ed638 100644
--- a/testtools/tests/test_testsuite.py
+++ b/testtools/tests/test_testsuite.py
@@ -181,7 +181,7 @@ class TestConcurrentStreamTestSuiteRun(TestCase):
 Traceback (most recent call last):
   File "...testtools/testsuite.py", line 188, in _run_test
     test.run(process_result)
-TypeError: run() takes ...1 ...argument (2 given)
+TypeError: run() takes ...1 ...argument...2...given...
 """, doctest.ELLIPSIS))
         events = [event[0:10] + (None,) for event in events]
         events[1] = events[1][:6] + (None,) + events[1][7:]
